### ISSUE TITLE
Export - import consistency issues between Newman and Postman client(s)

### ISSUE THREAD

**Issue by kunagpal** [postmanlabs/newman#573]
@czardoz @shamasis @numaanashraf @abhijitkane 

Related issues: #594, #569, #564, #553, #537, #389, #350, #289, #254, #196, #150, #102, #31 

Consider the following workflow:
1. Files are exported from the App.
2. The exported files are used by Newman and exported again.
3. The exported files are then used in two distinct ways:

a.  Used within Newman for a subsequent collection run: No variable resolution occurs, all assertions within the collection run fail. This is only valid for environment files, exported globals files are still re-usable, within Newman at least.

b. Imported within the Postman Client(s) (Chrome or Native): The client displays an error: `Failed to import data: Format not recognized`. This holds true for both: globals and environment files.

Sample environment exported by the Postman Client:
```json
{
	"id": "b8f07247-8238-a78a-c7d1-52686a35f279",
	"name": "export-test",
	"values": [
		{
			"key": "alpha",
			"value": "beta",
			"type": "text",
			"enabled": true
		},
		{
			"key": "gamma",
			"value": "delta",
			"type": "text",
			"enabled": true
		}
	],
	"timestamp": 1472208941298,
	"synced": false,
	"syncedFilename": "",
	"team": null,
	"isDeleted": false
}
```

Sample environment exported by Newman (notice the missing `id` and `name` keys):
```json
{
  "values": [
    {
      "key": "alpha",
      "name": "alpha",
      "value": "beta",
      "type": "string"
    },
    {
      "key": "gamma",
      "name": "gamma",
      "value": "delta",
      "type": "string"
    }
  ],
  "_postman_exported_at": "2016-08-26T11:12:50.776Z",
  "_postman_exported_using": "Newman/3.1.0"
}
```

**Comment by czardoz** [comment]
@kunagpal Can you add more details here? I want to see what Newman exports right now, and what the App expects.

**Comment by shamasis** [comment]
1. Let's first ensure all aspects of collection exported from current native apps work

2. Environments then (also globals)

3. Items exported from Newman should work in Newman!

**Comment by kunagpal** [comment]
Points one and two have seen no issues so far. As far as point three goes, the inconsistency in formats makes exported files non-reusable.

**Comment by rossmckelvie** [comment]
I ran into this issue. I exported the initial collections and environment from postman. The first collection runs to create a bunch of stuff and exports an updated environment file which I try to run with the second collection to delete them all.

The newman exported environment needs a `name` key at the root level, and each of the elements of `values` needs the `id` field renamed to `key`. I'm now able to use this exported file to run with the second collection.
